Mathlib/Util/Superscript.lean

```diff
@@ -204,15 +207,18 @@ def scriptParser.mapFormat (m : Mapping) (f : Format) : Except String Format :=
     .ok <| Std.Format.text ⟨s⟩
   | .align _ | .line | .nil => .ok f
 
-
 /-- Formatter for the script parser. -/
 def scriptParser.formatter (m : Mapping) (k : SyntaxNodeKind) (p : Formatter) : Formatter := do
   let stack ← modifyGet fun s => (s.stack, {s with stack := #[]})
   Formatter.node.formatter k p
   let s ← get
   match s.stack.mapM <| scriptParser.mapFormat m with
-  | .error s => throwError "Not a superscript: '{s}'"
-  | .ok newStack => set { s with stack := stack ++ newStack }
+  | .error err =>
+    -- TODO: this doesn't seem to go anywhere
+    Lean.logErrorAt (← get).stxTrav.cur s!"Not a superscript: '{err}'"
```

Comment:
maybe this should use a trace message instead?

Reply:
Do you have a trace class in mind?

Reply:
the usual one I enable when formatting is broken is `trace.PrettyPrinter.format.backtrack`, but you could also just use `trace.PrettyPrinter.format`

Reply:
I worked out how to make the error show up after all.